packages/api/src/controllers/taskController.js

```diff
@@ -24,6 +24,8 @@ import HarvestUse from '../models/harvestUseModel.js';
 import NotificationUser from '../models/notificationUserModel.js';
 import User from '../models/userModel.js';
 import { typesOfTask } from './../middleware/validation/task.js';
+import { Model, transaction } from 'objection';
```

Comment:
objection should only be imported inside the model files not in the controller file